config.yaml

```
@@ -13,39 +13,50 @@ output_dir: /output
 lab_structure:
   - title: Lab 1 - Operating System Perspective
     filename: lab1.md
-    content:
-      - modern-sw-stack.md
+    tasks:
       - basic-syscall.md
       - syscall-wrapper.md
       - libcall-syscall.md
+    content:
       - tasks/basic-syscall.md
       - tasks/syscall-wrapper.md
       - tasks/libcall-syscall.md
+      - modern-sw-stack.md
+      - basic-syscall.md
+      - syscall-wrapper.md
+      - libcall-syscall.md
   - title: Lab 2 - Library Perspective
     filename: lab2.md
     content:
-      - common-functions.md
-      - libc.md
-      - high-level-lang.md
-      - app-investigate.md
       - tasks/common-functions.md
       - tasks/libc.md
       - tasks/high-level-lang.md
       - tasks/app-investigation.md
+      - common-functions.md
```

This was deleted by mistake when `gen-view.py` was added.
```suggestion
      - guides/static-dynamic.md
      - common-functions.md
```